tutorials/parser/README.md

```diff
@@ -1,4 +1,27 @@
 This project contains applications showcasing different **Parsers** and **Formatters** present in the Malhar Library. For all the apps, parameters are configurable in META_INF/properties.xml.
 
 * **Json Parser App**  
-This app showcases **Json Parser**. Data generator sends Json data to the Json Parser which emits each record as POJO on the *output* port. The parser also has *parsedOutput* port that outputs each record as JSONObject and *error* port that emits a error records as key value pair. 
\ No newline at end of file
+This app showcases **Json Parser**. Data generator sends Json data to the Json Parser which emits each record as POJO on the *output* port. The parser also has *parsedOutput* port that outputs each record as JSONObject and *error* port that emits a error records as key value pair.
+
+* **Csv Parser App**
+
+This application showcases how to use [CsvParser](https://datatorrent.com/docs/apidocs/com/datatorrent/contrib/parser/CsvParser.html) from [Apex Malhar](https://github.com/apache/apex-malhar) library. The CsvParser converts your delimited data to a key-value pair map or concrete java class also know as [POJO](https://en.wikipedia.org/wiki/Plain_Old_Java_Object). The parser emits key-value pair map on *parsedOutput* port. It emits POJO on *out* and error records on *err* port.
+
+Follow these steps to run this application:
+
+
+**Step 1**: Update/add these properties in the file `src/main/resources/META-INF/properties-csvParseApplication.xml`:
+
```

Comment:
Why step 1 is necessary ?? I can see they are already there in the properties file. Can't i just build the app the run ?   Or do you want to say which properties users need to specify to use csv parser ? If so, its not clear .